lockbox-ios/View/StaticURLWebView.swift

```diff
@@ -38,5 +38,9 @@ class StaticURLWebView: UIViewController {
 
     private func styleNavigationBar() {
         self.navigationItem.title = self.navTitle
+
+        if #available(iOS 11.0, *) {
+            self.navigationItem.largeTitleDisplayMode = .never
```

Comment:
@sashei I hope you don't mind. I committed this here so the large title display from detail view didn't remain when loading the static URL view.

I was troubleshooting locally and had the change already in place so figured it was simpler for me to push a commit then ask you in a review. Feel free to back out if it's not the right result/approach though.